run_scratchblocks4.js

```diff
@@ -1,9 +1,23 @@
 function run_scratchblocks() {
-	scratchblocks.renderMatching('pre.blocks, pre[class^=blocks-3]', {languages: ['en'].concat(mw.config.get('wgScratchBlocks4Langs')), style: 'scratch3'});
-	scratchblocks.renderMatching('code.blocks, code[class^=blocks-3]', {languages: ['en'].concat(mw.config.get('wgScratchBlocks4Langs')), style: 'scratch3', inline: true});
-	scratchblocks.renderMatching('pre[class^=blocks-2]', {languages: ['en'].concat(mw.config.get('wgScratchBlocks4Langs')), style: 'scratch2'});
-	scratchblocks.renderMatching('code[class^=blocks-2]', {languages: ['en'].concat(mw.config.get('wgScratchBlocks4Langs')), style: 'scratch2', inline: true});
-	var items = document.querySelectorAll('.blocks .scratchblocks svg, [class^=blocks-3] .scratchblocks svg');
+	var version = mw.config.get('wgScratchBlocks4BlockVersion');
+	// Note: the weak equality is intentional to allow '2' and 2 both specify sb2
+	if (version == 2 || version[0] == '2') { // to handle '2.0'
+		version = 'scratch2';
+	} else {
+		version = 'scratch3';
+	}
+	var langs = ['en'].concat(mw.config.get('wgScratchBlocks4Langs'));
+	scratchblocks.renderMatching('pre.blocks', {languages: langs, style: version});
+	scratchblocks.renderMatching('code.blocks', {languages: langs, style: version, inline: true});
+	scratchblocks.renderMatching('pre[class^=blocks-3]', {languages: langs, style: 'scratch3'});
+	scratchblocks.renderMatching('code[class^=blocks-3]', {languages: langs, style: 'scratch3', inline: true});
+	scratchblocks.renderMatching('pre[class^=blocks-2]', {languages: langs, style: 'scratch2'});
+	scratchblocks.renderMatching('code[class^=blocks-2]', {languages: langs, style: 'scratch2', inline: true});
+	var query = '[class^=blocks-3] .scratchblocks svg';
```

Comment:
Just a little unclear on what's going on here

Reply:
s3blocks [need to be resized](https://github.com/InternationalScratchWiki/mw-ScratchBlocks4/pull/8/files/bed437c286d4f4e49d3af9b526f2ea515846aff8#diff-0ec7490e169d10ac15928460c38359b64fe6b5fe5d4e7f27db56dae7223b6c8dR25) and `query` selects all `.blocks-3*` svgs. If the default version (`version`) is scratch3 then generic `.blocks` svgs need to be included as well, which is done by the following `if` block.